doc/policies/admin.rst

```diff
@@ -391,12 +391,98 @@ type: bool
 To be able to define new :ref:`smtpserver` or delete existing ones, the
 administrator needs this rights ``smtpserver_write``.
 
+smtpserver_read
+~~~~~~~~~~~~~~~
+
+type: bool
+
+Allow the administrator to read the smtpserver configuration.
+
+smsgateway_write
+~~~~~~~~~~~~~~~~
+
+type: bool
+
+To be able to define new :ref:`smsgateway` or delete existing ones, the
+administrator needs this rights ``smsgateway_write``.
```

Comment:
typo: *the right

Reply:
will do

Reply:
done in 003f1a44
(might be a good idea to resolve conversations)